### PR TITLE
Add link to firmware recover instruction

### PR DESCRIPTION
**Pull request by KraPete**
изображение

Добавил ссылку на инструкцию по восстановлению прошивки, если нашли устройство в загрузчике

## Summary by CodeRabbit

- **New Features**
	- Added a new link to recovery instructions for devices in bootloader mode in both English and Russian translations.
	- Introduced a new error handling component that displays relevant messages when devices are in bootloader mode.

- **Bug Fixes**
	- Corrected spelling of the existing label from "in-bootloder" to "in-bootloader" in Russian translations.

- **Documentation**
	- Updated changelog to include new version (2.103.3) with enhancements for user guidance on firmware recovery.

### CODE REVIEW

File: app/scripts/react-directives/device-manager/scan/common.jsx [hunk 1]
```diff
@@ -56,6 +57,21 @@ const DeviceNameLabel = ({ title, selectable, selected, onSelectionChange, disab
...
+  }
+  if (selectable) {
+    return <ErrorTag text={t('scan.labels.in-bootloder')} />;
+  }
+  return (
+    <ErrorTag>
+      <Trans i18nKey={'scan.labels.in-bootloader-link'} components={[<a></a>]} />
+    </ErrorTag>
+  );
+};
```
[the remark this answers is not in the document]
Comment (KraPete) [reply app/scripts/react-directives/device-manager/scan/common.jsx:73]: Fixed

File: app/scripts/react-directives/device-manager/scan/common.jsx [hunk 2]
```diff
@@ -6,10 +6,11 @@ export function WarningTag({ text }) {
...
 }
 
-export function ErrorTag({ text, title }) {
+export function ErrorTag({ text, title, children }) {
   return (
     <span className="tag bg-danger text-nowrap" title={title}>
       {text}
+      {children}
     </span>
   );
```
[the remark this answers is not in the document]
Comment (KraPete) [reply app/scripts/react-directives/device-manager/scan/common.jsx:15]: useless

File: app/scripts/i18n/react/locales/ru/translations.json [hunk 3]
```diff
@@ -22,6 +22,7 @@
          "sn": "Серийный номер",
          "firmware": "Прошивка",
          "in-bootloder": "в загрузчике",
+         "in-bootloader-link": "в загрузчике, <0 href=\"https://wirenboard.com/wiki/Wb-mcu-fw-updater#%D0%92%D0%BE%D1%81%D1%81%D1%82%D0%B0%D0%BD%D0%BE%D0%B2%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5_(recover)\">инструкция по восстановлению</0>",
```
[the remark this answers is not in the document]
Comment (KraPete) [reply app/scripts/i18n/react/locales/ru/translations.json:25]: Fixed

File: debian/changelog [hunk 4]
```diff
@@ -1,3 +1,9 @@
+wb-mqtt-homeui (2.103.3) stable; urgency=medium
+
+  * Add link to firmware recover instruction
+
+ -- Petr Krasnoshchekov <petr.krasnoshchekov@wirenboard.com>  Wed, 06 Nov 2024 12:17:42 +0500
```
[the remark this answers is not in the document]
Comment (KraPete) [reply debian/changelog:5]: Fixed

File: app/scripts/i18n/react/locales/en/translations.json [hunk 5]
```diff
@@ -21,6 +21,7 @@
          "firmware": "Firmware",
          "available": "available {{version}}",
          "in-bootloder": "in bootloader",
+         "in-bootloader-link": "in bootloader, <0 href=\"https://wirenboard.com/wiki/Wb-mcu-fw-updater#%D0%92%D0%BE%D1%81%D1%81%D1%82%D0%B0%D0%BD%D0%BE%D0%B2%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5_(recover)\">how to recover</0>",
```
[the remark this answers is not in the document]
Comment (KraPete) [reply app/scripts/i18n/react/locales/en/translations.json:24]: Useless

File: app/scripts/react-directives/device-manager/scan/common.jsx [hunk 6]
```diff
@@ -56,6 +57,21 @@ const DeviceNameLabel = ({ title, selectable, selected, onSelectionChange, disab
...
+  }
+  if (selectable) {
+    return <ErrorTag text={t('scan.labels.in-bootloader')} />;
+  }
+  return (
+    <ErrorTag>
+      <Trans i18nKey={'scan.labels.in-bootloader-link'} components={[<a></a>]} />
+    </ErrorTag>
+  );
+};
```
[the remark this answers is not in the document]
Comment (KraPete) [reply app/scripts/react-directives/device-manager/scan/common.jsx:73]: Useless

**Review by sikmir — APPROVED** [review]

**Review by ekateluv — APPROVED** [review]

**Pull request merged**